app/logic/fileHandler.py

```diff
@@ -94,3 +94,9 @@ def deleteFile(self, fileId):
         if not AttachmentUpload.select().where(AttachmentUpload.fileName == file.fileName).exists():
             path = os.path.join(self.path, file.fileName)
             os.remove(path)
+    
+    def displayFile(self, fileId):
+        file = AttachmentUpload.get_by_id(fileId)
+        AttachmentUpload.update(isDisplayed=True).where(AttachmentUpload.id == fileId).execute()
+        AttachmentUpload.update(isDisplayed=False).where(AttachmentUpload.event == file.event, AttachmentUpload.id != fileId).execute()
```

Comment:
While it might not matter much in an application like ours, this is not very good practice. There is no reason to update every file in an event when, at most, we will  be unselecting 1 and selecting 1. So currently, instead of modifying the 2 files that are being changed, we are modifying all of them.